framework/src/main/java/org/checkerframework/common/value/ValueChecker.java

```diff
@@ -19,10 +19,21 @@
  * @checker_framework.manual #constant-value-checker Constant Value Checker
  */
 @StubFiles("statically-executable.astub")
-@SupportedOptions({ValueChecker.REPORT_EVAL_WARNS, ValueChecker.IGNORE_RANGE_OVERFLOW})
+@SupportedOptions({
+    ValueChecker.REPORT_EVAL_WARNS,
+    ValueChecker.IGNORE_RANGE_OVERFLOW,
+    ValueChecker.NULL_STRINGS_CONCATENATION
+})
 public class ValueChecker extends BaseTypeChecker {
+    /**
+     * Command-line option to warn the user if a @StaticallyExecutable method can't load and run at
+     * compile time.
+     */
     public static final String REPORT_EVAL_WARNS = "reportEvalWarns";
+    /** Command-line option to ignore the possibility of overflow for range annotations. */
     public static final String IGNORE_RANGE_OVERFLOW = "ignoreRangeOverflow";
+    /** Command-line option to allow elements of String concatenations to be null. */
```

Comment:
This doesn't allow them to be null, it assume that most expressions could be null.